src/machine/board_waveshare-rp2040-lora.go

```diff
@@ -0,0 +1,141 @@
+//go:build waveshare_rp2040_lora
+// +build waveshare_rp2040_lora
```

Comment:
Note: you don't need the `+build` lines anymore.

Reply:
that was a holdover from the other waveshare board I copied for this config. Will fix.